### PR TITLE
[WIP] Breadcrumb missions for random npcs

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Content "Breadcrumb missions for starter and random NPCs"

#### Purpose of change
Adds more ways to find the Hub01 and Refugee Center.

#### Describe the solution
Adds new random quests that can be picked by random NPCs.  Requires #49375

#### Describe alternatives you've considered
none

#### Testing
TBD

### CODE REVIEW

**Comment by I-am-Erk** [comment]
So, we could easily add a few random NPC backstories that mention different locations and add quests as a good start.

**Comment by Kanexan** [comment]
Just a heads-up, the Lapin quest-giver repeats the same acceptance/success/failure dialogues as for the Refugee Shelter; I don't think that's intentional.

File: data/json/npcs/BG_missions.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      "offer": "I hope you decide to help me find a safe place.",
+      "accepted": "Thank you.  Once you've found it come back and get me.",
+      "rejected": "You don't have to be mean about not being willing to help.",
+      "advice": "I've told you what I can remember of how to get there.",
+      "inquire": "Have you found the refugee center?",
+      "success": "What do you mean they won't take us in?",
+      "success_lie": "Well let's go then!",
+      "failure": "At least you came back with your life…"
+    }
+  },
```
Comment (Maleclypse) [inline data/json/npcs/BG_missions.json:105]:
```suggestion
  }
```

File: data/json/npcs/BG_missions.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    "value": 5000,
+    "origins": [ "ORIGIN_OPENER_NPC", "ORIGIN_ANY_NPC" ],
+    "//": "Update Origins to match background traits once that part is built.",
+    "dialogue": {
+      "describe": "I know the rumors say there is no government but surely that can't be true.",
+      "offer": "I hope you decide to help me find a safe place.",
+      "accepted": "Thank you.  Once you've found it come back and get me.",
+      "rejected": "You don't have to be mean about not being willing to help.",
+      "advice": "I've told you what I can remember of how to get there.",
+      "inquire": "Have you found the refugee center?",
```
Comment (Maleclypse) [inline data/json/npcs/BG_missions.json:74]:
```suggestion
      "inquire": "Are the stories true?",
```

File: data/json/npcs/BG_missions.json [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    "difficulty": 3,
+    "start": {
+      "assign_mission_target": { "om_terrain": "cabin_lapin", "om_special": "Cabin_Lapin", "reveal_radius": 3, "search_range": 360 }
+    },
+    "goal_condition": { "u_has_var": "u_met_lapin", "type": "dialogue", "context": "meeting", "value": "yes" },
+    "value": 5000,
+    "origins": [ "ORIGIN_OPENER_NPC", "ORIGIN_ANY_NPC" ],
+    "//": "Update Origins to match background traits once that part is built.",
+    "dialogue": {
+      "describe": "I know the rumors say there is no government but surely that can't be true.",
```
Comment (Maleclypse) [inline data/json/npcs/BG_missions.json:69]:
```suggestion
      "describe": "Strangest thing if he's survived all alone, while the world collapsed",
```

File: data/json/npcs/BG_missions.json [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+      "offer": "I hope you decide to help me find a safe place.",
+      "accepted": "Thank you.  Once you've found it come back and get me.",
+      "rejected": "You don't have to be mean about not being willing to help.",
+      "advice": "I've told you what I can remember of how to get there.",
+      "inquire": "Have you found the refugee center?",
+      "success": "What do you mean they won't take us in?",
```
Comment (Maleclypse) [inline data/json/npcs/BG_missions.json:75]:
```suggestion
      "success": "Did you get anything useful from the trip?",
```

**Comment by LyleSY** [comment]
Does #53894 adequately replace this?

**Comment by Maleclypse** [comment]
> Does #53894 adequately replace this?

It does. Thank you!

**Pull request closed without merging**